source/tests/test.js

```diff
@@ -325,3 +326,54 @@ describe('Multiple returns error validation', () => {
         assert.strictEqual(message, expected);
     });
 });
+
+describe('HTTP compression validation', () => {
+    it(`should return a compressed buffer for ads.txt using GZIP`, async () => {
+        const publicDir = './src/function/public';
```

Comment:
could be declared outside of subtests for all 3 or better to make it a class global as it is used multiple times.